REST API Docs.md

```
@@ -0,0 +1,1441 @@
+**Request Headers**
+
+| 헤더 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| Authorization | string | ✅ | Bearer {access_token} |
+
+**Response (200 OK)**
+
+```json
+
+{
+	"status": "success",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)